README.rst

```diff
@@ -23,6 +23,23 @@ The contents will be:
 - power colours
 - rms spectra
 
+Installation
+-------------
```

Comment:
I think we should say something about dependencies. E.g., one or more `conda` and/or `pip` commands that install all dependencies.